### DISCUSSION TITLE
Changing translation filename style

### DISCUSSION THREAD

**Discussion by ashhhleyyy** [WeblateOrg/weblate#7301] [Q&A] [answered]
I've been working on setting up Hosted Weblate for the NucleoidMC project, and as our translations are used within Minecraft mods, the translation file names should be in the form `en_us`/`fr_fr`/etc, and I've been struggling to figure out what to set the Language code style setting to correct this. An example of this is that [this file](https://github.com/NucleoidMC/plasmid/pull/210/files#diff-e4269883beb54b9d245c9838594a68646a6a2ef449ee626efc90539e1b8c1934) should be named `de_de.json` rather than just `de.json`.

Any help would be appreciated!

**Comment by nijel** [comment, marked as answer]
There is currently no support for creating lowercase only language codes, see https://docs.weblate.org/en/latest/admin/languages.html#new-translations
